apps/studio/pages/new/[slug].tsx

```diff
@@ -147,6 +147,12 @@ const Wizard: NextPageWithLayout = () => {
   const { data: organizations, isSuccess: isOrganizationsSuccess } = useOrganizationsQuery()
   const currentOrg = organizations?.find((o: any) => o.slug === slug)
 
+  // TODO: Remove this after project creation experiment
+  const projectCreationExperimentGroup = useFlag<string>('projectCreationExperimentGroup')
+  if (currentOrg && projectCreationExperimentGroup === 'group-b') {
+    router.push(`/new/v2/${currentOrg.slug}`)
```

Comment:
```suggestion
    router.replace(`/new/v2/${currentOrg.slug}`)
```
Using `.push()` here breaks the browsers back button. Would also mind popping this inside a `useEffect`? Half nit but more than anything makes me feel weird having right in render 😅